data/strings/types_strings.txt

```diff
@@ -11907,9 +11907,102 @@
   [type.emergency.lifeguard]
     comment = A place where a lifeguard is on duty.
     en = Lifeguard
+    en-GB = Lifeguard
+    af = Lewensredder
+    ar = منقذ
+    az = Xilasedici
+    be = Ратавальнік
+    bg = Спасител
+    ca = Socorrista
+    cs = Plavčík
+    da = Livredder
+    de = Rettungsschwimmer
+    el = Ναυαγοσώστης
+    es = Socorrista
+    es-MX = Socorrista
+    et = Päästja
+    eu = Soroslea
+    fa = نجات غریق
+    fi = Hengenpelastaja
+    fr = Sauveteur
+    he = מציל
+    hi = जीवन रक्षक
+    hu = Életmentő
+    id = Penjaga pantai
+    it = Bagnino
+    ja = ライフガード
+    ko = 인명 구조원
+    lt = Gelbėtojas
+    lv = Glābējs
+    mr = जीवरक्षक
+    nb = Livredder
+    nl = Strandwacht
+    pl = Ratownik
+    pt = Nadador-salvador
+    pt-BR = Guarda-vidas
+    ro = Salvamar
+    ru = Спасатель
+    sk = Plavčík
+    sr = Спасилац
+    sv = Livräddare
+    sw = Mlinzi wa maisha
+    th = ไลฟ์การ์ด
+    tr = Cankurtaran
+    uk = Рятувальник
+    vi = Nhân viên cứu hộ
+    zh = 救生员
+    zh-Hans = 救生员
+    zh-Hant = 救生員
 
   [type.emergency.mountain_rescue]
-    en = Mountain Rescue Station
+    comment = A rescue operation in mountainous terrain, usually to help hikers or climbers in distress.
```

Comment:
```suggestion
    comment = A mountain rescue base for search and rescue activities.
```
Its not just "Mountain Rescue" its exactly a station / base, so the `en = Mountain Rescue Station` is more correct.
ref: https://wiki.openstreetmap.org/wiki/Tag%3Aemergency%3Dmountain_rescue